### PR TITLE
Add GitHub Action Stale, remove Deprecated Probot Stale

### PR DESCRIPTION
**Pull request by juancarlospaco**
- https://github.com/probot/stale#-the-stale-app-is-deprecated-and-this-repository-is-no-longer-maintained
- https://github.com/probot/stale/issues/385#issuecomment-1555394911
- Using the official one instead, provided by GitHub https://github.com/actions/stale

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          days-before-pr-stale:    365
+          days-before-pr-close:    30
+          days-before-issue-stale: 1825  # 365 * 5
+          days-before-issue-close: 90
```
Comment (ringabout) [inline .github/workflows/stale.yml]: No issues please

**Review by ringabout — CHANGES_REQUESTED** [review]

File: .github/workflows/stale.yml (hunk 1 above)
Comment (ringabout) [reply .github/workflows/stale.yml]: The rest part is great.

Comment (ringabout) [inline .github/workflows/stale.yml]:
```suggestion
```

**Review by ringabout — APPROVED** [review]

File: .github/workflows/stale.yml (hunk 1 above)
Comment (juancarlospaco) [reply .github/workflows/stale.yml]: Can we instead use a huge value like `365 * 10` ?. 🤔

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            This pull request is stale because it has been open for 1 year with no activity.
+            Contribute more commits on the pull request and rebase it on the latest devel,
+            or it will be closed in 30 days. Thank you for your contributions.
+          close-pr-message: >
+            This pull request has been marked as stale and closed due to inactivity after 395 days.
+          stale-issue-message: >
+            This issue is stale because it has been open for 5 years with no activity.
+            Contribute a fix or comment on the issue, or it will be closed in 90 days.
+          close-issue-message: >
+            This issue has been marked as stale and closed due to inactivity after 1915 days.
```
Comment (juancarlospaco) [inline .github/workflows/stale.yml]:
```suggestion
```

**Pull request merged**